dockorm/tests/utils.py

```diff
@@ -54,7 +55,7 @@ def volume(path):
     Path to a file relative to the test volumes directory.
     """
     return join(
-        dirname(__file__),
+        getenv('DOCKORM_TESTS_DIR') or dirname(__file__),
```

Comment:
It looks like this matches the value used in https://github.com/quantopian/research_env/pull/560/files#diff-4e5e90c6228fd48698d074241c2ba760R14 . What makes this required?

Reply:
This is required only when running the tests themselves inside a docker container. 

The env var contains the path to the test volume data on the host. Since docker is running on the host, it will resolve the path for the volume being mounted to a path on the host (even when the request is coming from inside a container). 

Another option would be to ensure that the path inside the container matches that of the host, but that sounds challenging.